internal/rancher/management/v3/clusterregistrationtoken.go

```diff
@@ -0,0 +1,33 @@
+package v3
```

Comment:
is there reason for it to be v3?

Reply:
I guess Rancher objects are v3 so that is done to align with them